spec/models/user_spec.rb

```diff
@@ -272,4 +272,62 @@
       expect(described_class.random_password.scan(/[^A-Za-z0-9]/).count).to be >= 2
     end
   end
+
+  describe '.course_started?' do
+    subject(:user) { create(:user, :registered) }
+
+    context 'with no started modules' do
+      specify { expect(user.course_started?).to eq(false) }
```

Comment:
You have autogenerated RSpec matchers you might prefer, see how they compare.

Reply:
[20f17ef](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1031/commits/20f17ef9cd7f8599aff0818b0b48a539805322e1)

Reply:
Not what I was referring to but let me take look.